[2022-02-11 16:05:24,107][test_init_phone.py 15][INFO]: data===>[{'user_id': 1327}]
[2022-02-11 16:05:24,108][test_init_phone.py 17][INFO]: 1327
[2022-02-11 16:05:24,108][test_init_phone.py 21][INFO]: 随机数===>Yid2oLlf2F
[2022-02-11 16:05:24,108][test_init_phone.py 22][INFO]: 随机更新的手机号===>13656640242Yid2oLlf2F
[2022-02-11 16:05:24,108][test_init_phone.py 26][INFO]: ----------开始更新user表----------
[2022-02-11 16:05:24,351][mysql_operate.py 56][INFO]: 操作MySQL出现错误，错误原因：(1062, "Duplicate entry '13656640242Yid2oLlf2F' for key 'UK_OPEN_ID'")
[2022-02-11 16:05:24,474][test_init_phone.py 28][INFO]: ----------执行更新user表成功----------
[2022-02-11 16:05:24,475][test_init_phone.py 29][INFO]: ----------开始更新user_wechat_channel表----------
[2022-02-11 16:05:24,844][test_init_phone.py 31][INFO]: ----------执行更新user_wechat_channel表成功----------
[2022-02-11 16:05:24,845][test_init_phone.py 32][INFO]: ----------执行清除缓存的操作----------
[2022-02-11 16:05:25,248][redis_operate.py 21][INFO]: =====清除user_id为1327的token成功=====
[2022-02-11 16:05:25,374][redis_operate.py 22][INFO]: 2
[2022-02-11 16:05:25,741][redis_operate.py 25][INFO]: =====清除user_id为1327的tokens成功=====
[2022-02-11 16:05:25,741][test_init_phone.py 34][INFO]: ----------执行清除缓存操作成功----------
